code-mirror.yml

```diff
@@ -0,0 +1,19 @@
+trigger:
+    branches:
+        include:
+            - master
```

Comment:
Can we either remove master (doesn't seem to have been updated in 4 years) or rename it to v1.x (if that's why we're including it)

Reply:
Removed `master`